python/nano/test/tf/keras/test_trace_and_quantize.py

```diff
@@ -177,3 +177,19 @@ def test_inference_output_shape(self):
                                                       input_spec=tf.TensorSpec(shape=(None, 4)), x=x)
         outputs = quantized_model(x)
         assert isinstance(outputs, list) and isinstance(outputs[0], tf.Tensor)
+
+    def test_quantize_bf16(self):
```

Comment:
Maybe also test the case where there are customized layers?

Reply:
I don't think customized layer will work under this implementation. Customized layer will implement their own `get_config` and `from_config` method, they will set the dtye(if applied) in their `__init__` themselves. If users does not write any logic to set and change the internal dtype, then we could do nothing.

Reply:
yeah, that makes sense.